modules/installation-osp-kuryr-octavia-configuration.adoc

```diff
@@ -0,0 +1,201 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-installer-kuryr.adoc
+
+[id="installation-osp-kuryr-octavia-configuration_{context}"]
+= Configuring Octavia
+
+Kuryr SDN uses OpenStack Octavia LBaaS to implement OpenShift Services. Thus,
+you must install and configure Octavia components in your OpenStack environment
+to use Kuryr SDN.
+
+To enable Octavia, you must include the Octavia Service during the installation
+of the OpenStack Overcloud, or upgrade the Octavia Service if the Overcloud
+already exists. The following steps provide basic non-custom steps in enabling
```

Comment:
The wording is a little funny here. Maybe "The following non-custom steps for enabling Octavia apply to both..."

Reply:
Yeah, good call. I actually don't really like the `non-custom` word either. More explanation on the steps is given further down, so I'll remove that too.